### DISCUSSION TITLE
404 on kibana url since upgrade

### DISCUSSION THREAD

**Discussion by marcRBD** [Security-Onion-Solutions/securityonion#9786] [General] [answered]
hello everyone

Since i upgraded i have no kibana dashboard

https://url/kibana/
**"404 page not found"**

three nodes
SO
upgrage from Version: 2.3.110 to  2.3.210 

so-status

    Docker ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elastalert --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-filebeat ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-grafana ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-idstools ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-influxdb ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kratos ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-logstash ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-mysql -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-nginx -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-playbook ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-redis -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-sensoroni ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soc ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-wazuh -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

thanks for the help

**Comment by marcRBD** [comment, marked as answer]
it comes back 
after mutliple manipulation nothing.. but only a reboot of the manager did it

**Reply by cm-ops** [reply]
You were able to get Kibana working again?

**Reply by marcRBD** [reply]
yes, thank you
